packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -71,7 +64,7 @@ const { quote: q, details: X } = assert;
  * getLiquidityIssuer, and getLiquiditySupply.
  *
  * This contract has two parameters (poolFee and protocolFee) that are
- * managed by governance. The contract calls `handleParamGovernance()` at
+ * managed by governance. The contract calls `facetHelpers()` at
```

Comment:
Isn't it the case that the contract still calls handleParamGovernance?  These three changes to the comments may have been what confused me about the goal of this PR.